pkg/mimirpb/mimir.proto

```diff
@@ -156,6 +156,8 @@ message Histogram {
   ResetHint reset_hint               = 14;
   // timestamp is in ms format
   int64 timestamp = 15;
+
```

Comment:
Let's copy the comment from upstream: https://github.com/prometheus/prometheus/blob/de399eb09c90ba4e643b58bb998b0c158892d72e/prompb/types.proto#L111

Reply:
👍 done in https://github.com/grafana/mimir/pull/11007/commits/afe5171d09ac05aec3ebf900a15e32b1c2e4e7fc